pkgs/tools/misc/wacomtablet-kcm/default.nix

```diff
@@ -0,0 +1,32 @@
+{
+  stdenv, mkDerivation, fetchurl, extra-cmake-modules, qtx11extras,
+  plasma-workspace, libwacom, xf86_input_wacom
+}:
+
+mkDerivation {
+  name = "wacomtablet-kcm";
+  version = "3.2.0";
+  src = fetchurl {
+    url = "mirror://kde/stable/wacomtablet/3.2.0/wacomtablet-3.2.0.tar.xz";
+    sha256 = "197pwpl87gqlnza36bp68jvw8ww25znk08acmi8bpz7n84xfc368";
+    name = "wacomtablet.3.2.0.tar.xz";
```

Comment:
Why is this necessary?

Reply:
The `name` bit? It probably isn't necessary, just got copied and pasted from something which did require it.

Reply:
I think it should be removed if it's unnecessary. It adds another copy of the name and version